cfgov/deployable_zipfile/create.py

```diff
@@ -73,8 +72,6 @@ def create_zipfile(
         wheel_dir = os.path.join(temp_dir, "wheels")
 
         wheel_executables = [sys.executable]
```

Comment:
The `for` loop is no longer needed, and this whole code block (lines 75-82) can be simplified to

```py
save_wheels(sys.executable, wheel_dir, "-r%s" % requirements_file)
```

Reply:
Removed in [84918c2](https://github.com/cfpb/consumerfinance.gov/pull/7078/commits/84918c2b666993d4b1edc9a43959dd8288041a36).